src/main/java/org/isf/sms/service/SmsSender.java

```diff
@@ -57,77 +56,50 @@ public void run() {
 			List<Sms> smsList = null;
 			try {
 				smsList = smsOp.getList();
-			} catch (OHServiceException ohServiceException) {
-				LOGGER.error(ohServiceException.getMessage(), ohServiceException);
+			} catch (OHServiceException e1) {
```

Comment:
Logging exceptions with message and trace are the standard; not just printStackTrace()